### DISCUSSION TITLE
NicTool 3.0

### DISCUSSION THREAD

**Discussion by msimerson** [NicTool/NicTool#272] [Ideas]
- How can NicTool become a better product?
- Feature requests?
    - #286 
    - #275 
- SaaS option?

**Comment by meersau** [comment]
+1 for JSON API (REST)

**Comment by rduffner** [comment]
Recently, SWITCH (the Swiss DNS registry) started to charge more for DNSSEC-less domains.

So DNSSEC would be plus.
